[2025-07-02T08:57:07.089+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-02T08:57:07.244+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-07-02T08:56:51.686460+00:00 [queued]>
[2025-07-02T08:57:07.294+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-07-02T08:56:51.686460+00:00 [queued]>
[2025-07-02T08:57:07.296+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-07-02T08:57:07.363+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-07-02 08:56:51.686460+00:00
[2025-07-02T08:57:07.473+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=792) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-02T08:57:07.543+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'manual__2025-07-02T08:56:51.686460+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmpv3mz7r47']
[2025-07-02T08:57:07.561+0000] {standard_task_runner.py:72} INFO - Started process 801 to run task
[2025-07-02T08:57:07.573+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask extract_raw_data
[2025-07-02T08:57:08.935+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-07-02T08:56:51.686460+00:00 [running]> on host 5e24963c1987
[2025-07-02T08:57:09.354+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-02T08:56:51.686460+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-02T08:56:51.686460+00:00'
[2025-07-02T08:57:09.363+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-02T08:57:09.501+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-02T08:57:09.567+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-02T08:57:09.570+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=manual__2025-07-02T08:56:51.686460+00:00, execution_date=20250702T085651, start_date=20250702T085707, end_date=20250702T085709
[2025-07-02T08:57:09.737+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-02T08:57:09.917+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-02T08:57:09.929+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
